front/lib/api/workspace.ts

```diff
@@ -9,6 +9,15 @@ import {
 import { MembershipInvitationType } from "@app/types/membership_invitation";
 import { UserType } from "@app/types/user";
 
+export async function isWorkspaceAllowedOnDomain(wId: string) {
```

Comment:
Is that okay to add a function in lib without an auth object? In this specific case we won't have one as there's no user logged-in, so I'm returning only a boolean and no other workspace information.

Reply:
Yes makes sense :+1: